web/src/components/deployments-detail-page/pipeline/index.tsx

```diff
@@ -283,6 +315,21 @@ export const Pipeline: FC<PipelineProps> = memo(function Pipeline({
             </Button>
           </DialogActions>
         </Dialog>
+
+        <Dialog open={isOpenSkipDialog} onClose={() => setSkipTargetId(null)}>
+          <DialogTitle>Skip stage</DialogTitle>
+          <DialogContent>
+            <DialogContentText>
+              {`To skip this stage, click "SKIP".`}
+            </DialogContentText>
+          </DialogContent>
+          <DialogActions>
+            <Button onClick={() => setSkipTargetId(null)}>CANCEL</Button>
```

Comment:
FIXME: I wanna make it to hide the dialog and show the process log, but now we can just only hide the dialog.
I wonder how to call the both of `setSkipTargetId(null)` and `handleOnClickStage` at the same onClieck.

Reply:
Addressed by showing the process log when clicking the `ANALYSIS`.